Problem2/largest_two_numbers.rb

```diff
@@ -0,0 +1,29 @@
+# File: largest_two_numbers.rb
+
+class LargestTwoNumbers
```

Comment:
Oh, nooo... documentation :) The agony!
Just get creative
Here's my one liner :)
```
def get_largest_two_numbers(array)
  [array.delete(array.max), array.delete(array.max)]
end
```
one possibility is, now it's taken so you cannot do this:

I'm just curious how you discover ruby.